aries-site/src/pages/templates/app-layouts.mdx

```diff
@@ -44,8 +62,8 @@ The Page Container is responsible for the layout for the main content of a page.
 It contains a page's [Page Header](/components/header#header-for-a-single-page) 
 and its Page Content.
 
-### Container Width
-There are 3 content width options from which to choose, depending on 
+### Page Container Widths
+There are 3 Page Container widths from which to choose, depending on 
 the layout use case: [Wide](#wide-default), [Narrow](#narrow) and [Full](#full). 
 
 In all cases, the minimum width of content is `336px`, after which horizontal 
```

Comment:
Is this a t-shirt size, or where does this derive from?

Reply:
Good question. These are some of the things we want to test out to see how design direction translates into dev implementation. What disconnects are there, if any? What are the friction points? ....

In theory, 336 + 24 + 24 = 384 (medium Box). So I'd like to try to implement a few screens at these breakpoints and see where things work/breakdown. That way we can identify any issues in the translation from theory to implementation, have a conversation and then we can iterate from there.